src/panels/ValidationPanel.tsx

```diff
@@ -123,92 +114,98 @@ export default class ValidationPanel extends React.Component<Props, State> {
+            <div className={"centeredValidation"}>
+              {Locale[AppSettings.viewLanguage].validationLoadingError}
+            </div>
+          )}
+          {this.state.loading && (
+            <div className={"centered"}>
+              <Spinner animation={"border"} />
+            </div>
+          )}
+          {!this.state.loading && !this.state.conforms && !this.state.error && (
+            <div style={{ overflow: "auto", height: "inherit" }}>
+              <TableList
+                headings={[
+                  Locale[AppSettings.viewLanguage].validationNumber,
+                  Locale[AppSettings.viewLanguage].validationSeverity,
+                  Locale[AppSettings.viewLanguage].validationName,
+                  Locale[AppSettings.viewLanguage].validationError,
+                ]}
+              >
+                {this.state.results.map((result, i) => (
+                  <tr>
+                    <td>
+                      <button
+                        className={"buttonlink"}
+                        onClick={() => this.focus(result.focusNode)}
+                      >
+                        {i + 1}
+                      </button>
+                    </td>
+                    <td>
+                      {result.severity.substring(
+                        result.severity.lastIndexOf("#") + 1
+                      )}
+                    </td>
+                    {result.focusNode in WorkspaceTerms ? (
+                      <IRIlabel
+                        label={
+                          WorkspaceTerms[result.focusNode].labels[
+                            this.props.projectLanguage
+                          ]
+                        }
+                        iri={result.focusNode}
+                      />
+                    ) : (
+                      <IRIlabel
+                        label={result.focusNode}
+                        iri={result.focusNode}
+                      />
                     )}
-                  </td>
-                  {result.focusNode in WorkspaceTerms ? (
-                    <IRIlabel
-                      label={
-                        WorkspaceTerms[result.focusNode].labels[
-                          this.props.projectLanguage
-                        ]
-                      }
-                      iri={result.focusNode}
-                    />
-                  ) : (
-                    <IRIlabel label={result.focusNode} iri={result.focusNode} />
-                  )}
-                  <td>
-                    {result.message.includes("@")
-                      ? result.message.substring(
-                          0,
-                          result.message.lastIndexOf("@")
-                        )
-                      : result.message}
-                  </td>
-                </tr>
-              ))}
-            </TableList>
-          </div>
-        )}
+                    <td>
+                      {result.message.includes("@")
```

Comment:
It would be best to extract this substring code to a separate helper function, but it's just a minor remark.